.github/workflows/collector.yml

```diff
@@ -66,8 +76,19 @@ jobs:
           collector_image: ${{ inputs.collector-image }}
           collector_tag: ${{ inputs.collector-tag }}
           debug_mode: ${{ github.event_name == 'pull_request' }}
+          driver_version: ${DRIVER_VERSION}
+          context_path: ${{ github.workspace }}/kernel-modules/container
```

Comment:
Bit puzzling that this is not `${{ env.CONTEXT_DRIVERS_DIR }}`, can we unify them somehow?

Reply:
Missed this when I was fiddling around :) I've removed it now